tests/components/tessie/test_coordinator.py

```diff
@@ -79,3 +85,48 @@ async def test_coordinator_connection(hass: HomeAssistant, mock_get_status) -> N
     await hass.async_block_till_done()
     mock_get_status.assert_called_once()
     assert hass.states.get("binary_sensor.test_status").state == STATE_UNAVAILABLE
+
+
+async def test_coordinator_live_error(hass: HomeAssistant, mock_live_status) -> None:
+    """Tests that the energy live coordinator handles fleet errors."""
+
+    await setup_platform(hass, [Platform.SENSOR])
+
+    mock_live_status.reset_mock()
+    mock_live_status.side_effect = Forbidden
+    async_fire_time_changed(hass, utcnow() + TESSIE_FLEET_API_SYNC_INTERVAL)
+    await hass.async_block_till_done()
+    mock_live_status.assert_called_once()
+    assert hass.states.get("sensor.energy_site_solar_power").state == STATE_UNAVAILABLE
+
+
+async def test_coordinator_info_error(hass: HomeAssistant, mock_site_info) -> None:
+    """Tests that the energy info coordinator handles fleet errors."""
+
+    await setup_platform(hass, [Platform.SENSOR])
+
+    mock_site_info.reset_mock()
+    mock_site_info.side_effect = Forbidden
+    async_fire_time_changed(hass, utcnow() + TESSIE_FLEET_API_SYNC_INTERVAL)
+    await hass.async_block_till_done()
```

Comment:
```suggestion
    async_fire_time_changed(hass)
    freezer.tick(TESSIE_FLEET_API_SYNC_INTERVAL)
    await hass.async_block_till_done()
```
Please use the freezer when changin time